### ISSUE TITLE
"\emoji heavy_check_mark" produces ✓ instead of  ✔️

### ISSUE THREAD

**Issue by asiaclonk** [doxygen/doxygen#8169]
**Describe the bug**
```\emoji heavy_check_mark```
is parsed into this character: ✓

**Expected behavior**
It should parse into ✔️ instead, as shown in multiple lists.

**Workaround**
It is possible to bypass this by directly pasting ✔️ without the \emoji command into the documentation.

**Version**
Doxygen 1.8.20 on Windows 10 64-bit.
Documentation built using the Visual Studio plugin, run on C# code, output as html.
Witnessed on the Chrome Browser.

**Comment by albert-github** [comment]
You wrote:
> It should parse into ✔️ instead, as shown in multiple lists.

- which lists?

The representation of the different emoji is always depending on the used display software, doxygen translates the `heavy_check_mark` into the HTML code `&#x2714` (based on the Github list https://api.github.com/emojis for images and the code number) and leaves in this case the display to the browser).
It also depends on the used font family (as can be seen on https://unicode.org/emoji/charts/full-emoji-list.html as `font-family: "Noto Color Emoji", "Apple Color Emoji", "Segoe UI Emoji", Times, Symbola, Aegyptus, Code2000, Code2001, Code2002, Musica, serif, LastResort;`.

I think the the problem is not really a doxygen bug, but it would probably be nice when there is a possibility for an emoji to set the proper font family,

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8170

**Comment by asiaclonk** [comment]
> which lists?

I was referring to this link https://gist.github.com/rxaviers/7360908 in particular, which is referred to in the official documentation for the \emoji command in https://www.doxygen.nl/manual/commands.html#cmdemoji
I remember finding a more complete list somewhere else, which also listed heavy_check_mark with the expected emoji.

**Comment by albert-github** [comment]
The markdown processor of github must do something special in respect to translating an emoji. When I use the name of the emoji I get on github:
 :heavy_check_mark: `:heavy_check_mark:`
and when I use the unicode version (as doxygen uses):
 &#x2714; `&#x2714;`
I tried to disable the font family on github but this has no effect, either github uses somewhere some hidden setting or it uses an image (I could not, yet, find it cause).
I think the proposed solution gives the user a flexible possibility and remain compatible with the released versions.


TEST:
:white_flower: `:white_flower:`

**Comment by albert-github** [comment]
I think I got it on github, here a special tag is used `g-emoji` and in the code we see:
```
<g-emoji class="g-emoji" alias="heavy_check_mark" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/2714.png">✔️</g-emoji>
```
and there is: https://github.com/github/g-emoji-element/blob/main/README.md documentation.
Most likely this package does something special.

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
